packages/commons/src/events/FieldConfig.ts

```diff
@@ -413,6 +418,15 @@ const PhoneField = BaseField.extend({
   type: z.literal(FieldType.PHONE)
 }).describe('Phone input field')
 
+const IntlPhoneField = BaseField.extend({
+  country: z.string(),
```

Comment:
Let's move this inside `configuration` as that's the convention to separate field-specific configuration from general field configurations. 

```
configuration.defaultCountry
```

works